### ISSUE TITLE
Workspace switcher applet does not accept subsequent mouse-wheel inputs whilst transitioning

### ISSUE THREAD

**Issue by Reposium** [budgie-desktop/budgie-desktop#1118]
**Operating System**: Solus 3
**Budgie version**: Budgie 10.4

**The issue encountered**
If I switch workspaces using the mouse-wheel on the workspace switcher applet, all subsequent mouse-wheel inputs are ignored until the workspace switch animation completes. Using keyboard shortcuts, I can switch between workspaces even whilst Budgie is still animating a previous change. 

This only occurs for mouse-wheel inputs after a workspace switch that was caused by an initial mouse-wheel switch. If I change workspaces using keyboard shortcuts/mouse-click, I can switch desktops mid-animation using mouse-wheel on the switcher. However, no furthur workspace changes are possible with mouse-wheel until after the animation for this switch (initiated by mouse-wheel) is complete.

To sum up:  
- switch with mouse-wheel: cannot switch with mouse-wheel until first switch complete
- switch mouse-click/keyboard shortcuts: can switch with mouse-wheel, even if mid-animation

**Comment by Reposium** [comment]
On the other hand, I do realise that this behaviour makes switcher applet usable when using a laptop trackpad or one of those apple mice with the smooth scroll wheel, and allows one to flick any scrolling device without worrying about zooming 3 workspaces too far. Feel free to close this if this is intended.

**Comment by Tito1337** [comment]
I agree with @Reposium : there is no need for such a long delay before allowing the next workspace change.

This shoud be easy to fix, as it is an explicit timeout in the code : https://github.com/budgie-desktop/budgie-desktop/blob/9f71bb847ef73a14e2cd248b34aaa3866b7e795c/src/applets/workspaces/WorkspacesApplet.vala#L178-L180
